CONTRIBUTING.md

```diff
@@ -28,45 +23,37 @@ We keep the generated fonts under version control, so the last step is to run:
 
 Which will build the `.otf` font files that should be committed as well.
 
-Fonts must be generated for each commit that changes the source files. The
-build tools are smart enough to not change the binary fonts if the sources were
-not changed.
+Fonts must be generated for each commit that changes the source files.
+The build tools are smart enough to not change the binary fonts if the sources were not changed.
 
-### Optional dependencies
+## Optional dependencies
 
-The build process, besides generating the `.otf` font files, also updates the
-SVG and PDF preview documents. For that, additional tools are needed: `xelatex`
-from [xetex][], and `mutool` from [mupdf][] (version 1.15.0 or above). The
-`make` command will emit errors if these are not present, but as long as the
-font generation stages finish correctly, these errors can be ignored. However,
-if you do wish to update the preview files, these tools must also be installed.
+The build process, besides generating the `.otf` font files, also updates the SVG and PDF preview documents.
+For that, additional tools are needed: `xelatex` from [xetex][], and `mutool` from [mupdf][] (version 1.15.0 or above).
+The `make` command will emit errors if these are not present, but as long as the font generation stages finish correctly, these errors can be ignored.
+However, if you do wish to update the preview files, these tools must also be installed.
 
-#### Linux
+### Linux
 
-On most Linux distros, the `xelatex` binary is provided by the
-[`texlive-xetex` package][repology-xetex], and the `mutool` binary by
+On most Linux distros, the `xelatex` binary is provided by the [`texlive-xetex` package][repology-xetex], and the `mutool` binary by
 [`mupdf-tools` or `mupdf`][repology-mupdf].
```

Comment:
I suspect you may have missed this line break, @alerque.

Reply:
Good catch.